apps/opik-backend/src/main/java/com/comet/opik/OpikApplication.java

```diff
@@ -69,7 +70,7 @@ public void initialize(Bootstrap<OpikConfiguration> bootstrap) {
                         .withPlugins(new SqlObjectPlugin(), new Jackson2Plugin()))
                 .modules(new DatabaseAnalyticsModule(), new IdGeneratorModule(), new AuthModule(), new RedisModule(),
                         new RateLimitModule(), new NameGeneratorModule(), new HttpModule(), new EventModule(),
-                        new ConfigurationModule(), new BiModule())
+                        new ConfigurationModule(), new BiModule(), new EncryptionUtilsModule())
```

Comment:
We don't need a module since this will be static. We can just set the configuration using the `run` method below.

Reply:
Yes, it could be set in run. Since we are using DI this approach seems more consistent. Can be changed, don't have any preferences here.